### ISSUE TITLE
Adding Seed to Wildcards Nodes - Regenerate same choices

### ISSUE THREAD

**Issue by DeepSeaCatz** [chibiace/ComfyUI-Chibi-Nodes#9]
Hi there,

I've been loving your Wildcards node and made a workflow that generate random characters:
[https://civitai.com/models/572534/random-character-generator](https://civitai.com/models/572534/random-character-generator)

I'm currently working on adding further details like a Face Detailer and an Upscaler. The thing is that the current workflow works super fast to generate random characters, but takes a lot more time when going through the Face Detailer + Upscaler. I thought it'd be interesting if there was a way to keep generating and then when one generation is interesting enough to add more details that it would be nice to have all the same wildcards choices (workflow included) :
![RandomCharacterGenerator_FaceDetailer](https://github.com/user-attachments/assets/b8bdb4f3-fd74-44a9-8643-dc52dcdfebfd)

This made me think that it could be nice if your Wildcards node would have a Seed input. This would let us choose the same choice that it did before from the txt file. I could use a "Seed Everywhere" node to connect every Wildcards and keep it at randomize generation and then set the seed to Fixed with the same number to generate the same wildcards choice:

![image](https://github.com/user-attachments/assets/c59ab936-2866-4657-ae3f-8980d08a3ae2)

That being said, I'm not sure if your node would be compatible with seed or if you had another suggestion for this issue.

Thank you.

**Comment by chibiace** [comment]
at the moment it is getting its randomness from the python builtin random library and i believe its seed is based off the system time, i might be able to fix something up so if an integer is connected to a seed input it uses that number. 


i'll try to find time to look into adding this over the next week or so.

**Comment by DeepSeaCatz** [comment]
Yeah adding an if statement for an integer input would work wonderfully! Thank you for taking the time to implement this. It will be very useful to gain more control in wildcards, mostly from a node that chooses from a txt list.

**Comment by chibiace** [comment]
alright added the seed input which takes an integer, im thinking to do a total rewrite after having to read my own code. (╯°□°)╯︵ ┻━┻

**Comment by DeepSeaCatz** [comment]
Awesome! Works super well, thank you so much! I've combined this with a Seed Everywhere node and it re-generate the same prompts with the same seed now.

I've added a Face Detailer + Upscaler to my workflow. I'll update it soon after a couple of test with the new Seed of your Wildcards. [https://civitai.com/models/572534/random-character-generator](https://civitai.com/models/572534/random-character-generator)

I'll keep an eye if you ever re-write the code so I can test it. This just means you've improved your coding skills 😄

Thanks again!
